website/docs/reference/project-configs/log-path.md

```diff
@@ -47,12 +47,17 @@ The precedence order is: CLI flag > env var > `dbt_project.yml`
 </VersionBlock>
 
 ## Examples
-### Write logs to a subdirectory named `dbt_logs` instead of `logs`
+### Specify subdirectory using the project config file
 
 <File name='dbt_project.yml'>
 
 ```yml
 log-path: dbt_logs
 ```
 
+### Specify subdirectory from the command line 
```

Comment:
once i get the lists, i can add CLI examples like i did in this file. thanks!

Reply:
does it make sense to add more examples here with the sub-command ordering changes? the GH issue mentioned it so am just following up on that bit

Reply:
@dbeatty10 What are your thoughts here? Any examples that you think we should call out?

Reply:
`--log-path` was the flag mentioned [here](https://github.com/dbt-labs/dbt-core/issues/7764) and [here](https://getdbt.slack.com/archives/C37J8BQEL/p1683214167445399?thread_ts=1682627562.742869&cid=C37J8BQEL), so I think the example below will suffice ✅ 

`--target-path` and `profiles-dir` are examples of flags that must appear _after_ the subcommand.